crates/server/src/certificates.rs

```diff
@@ -0,0 +1,164 @@
+use rcgen::{CertificateParams, DnType};
+use local_ip_address::local_ip;
+use std::fs;
+use std::path::Path;
+use x509_parser::prelude::*;
```

Comment:
how many things would you have to pull in to avoid this blanket import? let's favor explicitness, unless with good reason